packages/components/app/styles/components/dropdown.scss

```diff
@@ -21,31 +21,21 @@
 
 @use '../mixins/focus-ring' as *;
 
-$hds-dropdown-toggle-base-height: 36px;
-$hds-dropdown-toggle-border-radius: 5px;
-$hds-dropdown-toggle-border-width: 1px;
```

Comment:
@MelSumner I didn't suggest to remove all the variables, just the last one (was used only once).

Reply:
https://github.com/hashicorp/design-system/pull/202